package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "license": "UNLICENSED",
+  "workspaces": [
+    "packages/*"
+  ],
+  "resolutions": {
+    "@polkadot/api": "^5.8.3",
+    "@polkadot/types": "^5.8.3"
+  },
```

Comment:
What does this "resolutions" field do?

Reply:
It mades that all dependencies using these packes use the vesion declared in here.

Reply:
Removed at: 162d1a2